task-launcher/src/styles/task.scss

```diff
@@ -625,8 +625,17 @@ $breakpoint-large-desktop: 1440px;
       .btn-layout-hf {
```

Comment:
This is the only change that impacts a file common to the other tasks. 
As indicated by the suffix '-hf' it is to my knowledge not used outside of the Hearts and Flowers task code.